README.md

```diff
@@ -1,51 +1,110 @@
 # section-metadata-wg-onto
 
-This repository is meant to organize the work of the [NFDI Section (Meta)Data](https://www.nfdi.de/section-metadata/) Working Group Ontology harmonization and Mapping.
+This repository is meant to organize the work of the [NFDI Section (Meta)Data](https://www.nfdi.de/section-metadata/) Working Group Ontology harmonization 
+and Mapping.
+
+This repository is two things. First, it is the cloud storage space where we upload and share files with our WG
+members and the rest of the world in a version controlled manner. Second, it is the space in which we can discuss 
+asynchronously and transparently the tasks we need to accomplish via GitHub Issues and Pull Requests.
+
+## Prerequisites
```

Comment:
Rename section to "Contributing"?

Reply:
Done in [3f3c950](https://github.com/nfdi-de/section-metadata-wg-onto/pull/16/commits/3f3c9500c85fb01c79a59415098334f60b427683)